web/src/Pages/TopicPage/TopicCard/TopicCard.js

```diff
@@ -29,3 +31,13 @@ export default function TopicCard({ topic }) {
     </div>
   );
 }
+
+TopicCard.propTypes = {
+  topic: PropType.shape({
+    name: PropType.string,
+    description: PropType.arrayOf(PropType.object),
+    parent: PropType.object,
+    relationships: PropType.arrayOf(PropType.object),
```

Comment:
I have typed out the shape of the object for the relationships and when I tested the your branch on my device it seems to be working fine. Do try it on yours and if it gives warnings in console, just stick to `PropType.object`.
```suggestion
    relationships: PropType.arrayOf(PropType.shape({
        audience: PropType.number.isRequired,
        description: PropType.string.isRequired,
        to: PropType.shape({
          image: PropType.string.isRequired,
          name: PropType.string.isRequired,
          slug: PropType.string.isRequired,
        }).isRequired,
      }).isRequired,),
```